### PR TITLE
fix(code-style): polish the logs/logs.go

### PR DESCRIPTION
**Pull request by donggangcj**
1. private method statement use lower case
2. remove the meaningless bracket
...

Signed-off-by: Dong Gang <dong.gang@daocloud.io>

### CODE REVIEW

File: src/app/backend/resource/logs/logs.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 )
 
 // LINE_INDEX_NOT_FOUND is returned if requested line could not be found
-var LINE_INDEX_NOT_FOUND = -1
+var LineIndexNotFound = -1
```
Comment (maciaszczykm) [inline src/app/backend/resource/logs/logs.go:23]: It does not match anymore...

File: src/app/backend/resource/logs/logs.go [hunk 2]
```diff
@@ -251,7 +251,7 @@ func ToLogLines(rawLogs string) LogLines {
 	logLines := LogLines{}
 	for _, line := range strings.Split(rawLogs, "\n") {
 		if line != "" {
-			startsWithDate := ('0' <= line[0] && line[0] <= '9') //2017-...
+			startsWithDate := '0' <= line[0] && line[0] <= '9' //2017-...
```
Comment (maciaszczykm) [inline src/app/backend/resource/logs/logs.go:254]: It's an only cosmetic change. This is not required.

File: src/app/backend/resource/logs/logs.go [hunk 3]
```diff
@@ -216,10 +216,10 @@ func (self LogLines) getLineIndex(logLineId *LogLineId) int {
 	if 0 <= offset && offset < linesMatched {
 		return matchingStartedAt + offset
 	}
-	return LINE_INDEX_NOT_FOUND
+	return LineIndexNotFound
 }
 
-// CreateLogLineId returns ID of the line with provided lineIndex.
+// createLogLineId returns ID of the line with provided lineIndex.
```
Comment (maciaszczykm) [inline src/app/backend/resource/logs/logs.go:222]: +1

File: src/app/backend/resource/logs/logs.go (hunk 1 above)
Comment (donggangcj) [reply src/app/backend/resource/logs/logs.go:23]: I got it.Thanks!

**Comment by maciaszczykm** [comment]
Reopen after fixing the mentioned issues.

File: src/app/backend/resource/logs/logs.go (hunk 2 above)
Comment (donggangcj) [reply src/app/backend/resource/logs/logs.go:254]: Yeap! this is from a IDE format warning.

**Comment by maciaszczykm** [comment]
/lgtm

It's okay. Codecov is not working well with smaller changes.

**Pull request merged**